README.md

```diff
@@ -61,6 +128,30 @@ php bin/magento cache:clean
     >`http://[your-site-url]/snap/index/finish`
 
 
+
+## How to online refund transaction
+
+You can request refunds either in the [Midtrans Dashboard](https://dashboard.midtrans.com/transactions) or in the Magento admin. After a refund is issued, it cannot be cancelled or undone. Before you trigger this request, make sure that the refund amount and any other details are correct. The online refund function is available only for payment method gopay and credit card.
+
+If you make a refund in the Midtrans Dashboard, a Refund notification is sent to Magento, set transaction state to CLOSE and for now is not created the credit memo.
+
+
+### Request refund from Magento Admin:
+
+1. Log in to your Magento admin panel. 
+2. In the menu, go to **Sales** > **Orders**. This opens the order overview page. 
+3. Click on the order you want to refund.
+4. In the **Order list View** left-hand navigation sidebar, click **Invoices**tab.
+5. In the invoice list page, selected the order. click the view button on invoice you need to request online refund.
+6. Click **Credit Memo** on the top-right corner of the page.
+7. In the **New Memo for Invoice** page, scroll down to the **Refund Totals** section.
+8. In this section, you can request online **Refund** or **Refund Offline**.
+    *   **Refund:** This option will request refund Online to Midtrans, Midtrans automatically send refund notification and changes order status to **Closed** from notification.
+    *   **Refund Offline**: An offline refund does not trigger request refund to midtrans it’s only refund in Magento side. You need to take action and carry out the refund manually from Midtrans dashboard.After a refund operation, the order status changes to **Closed**. This order status change is controlled by the Magento system. 
+    
+    The order status change does not mean that the refund was carried out successfully on Midtrans side. If the transaction status in Midtrans dashboard changes to **REFUND**, then the refund went through successfully
```

Comment:
Fix to: 
> The status change may not mean that the refund has carried out successfully on Midtrans side. When the transaction status in Midtrans dashboard changes to **REFUND**, then the refund went through successfully

Reply:
done

Reply:
What's `done`?
I asked to revise the sentence to 
> The status change may not mean that the refund has carried out successfully on Midtrans side. When the transaction status in Midtrans dashboard changes to REFUND, then the refund went through successfully

Why didn't it get revised? And instead you add new sentence with another grammar mistakes.

Reply:
should be in accordance with the request @rizdaprasetya , sorry missed for that